client/src/SimpleSnackbar.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import Button from '@material-ui/core/Button';
+import Snackbar from '@material-ui/core/Snackbar';
+import IconButton from '@material-ui/core/IconButton';
+import CloseIcon from '@material-ui/icons/Close';
+import CheckCircleIcon from '@material-ui/icons/CheckCircle'
+import {makeStyles, Theme} from '@material-ui/core/styles';
+
+
+const useStyles=makeStyles((theme:Theme)=>({
+    root:{
+        width: '100%',
+    },
+}))
+
+export default function SimpleSnackbar(){
+    const classes=useStyles()
+    const [open, setOpen]=React.useState(true)
+
+    const handleClick = () =>{
+        setOpen(true)
+    }
+    const handleClose = (event?: React.SyntheticEvent, reason?: string) =>{
+        if(reason=="clickaway"){
```

Comment:
this if block is doing nothing, did you mean to `return` in there so that we don't hit the `setIsOpen`?
if not let's remove

Reply:
also triple equals always plz